Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -231,6 +230,11 @@ public virtual bool CanLaunchWebBrowser
         /// </summary>
         public int HttpsPort { get; private set; }
 
+        /// <summary>
+        /// Gets the value of the PublishedServerUrl setting.
+        /// </summary>
+        public string PublishedServerUrl => _startupOptions.PublishedServerUrl ?? _startupConfig["PublishedServerUrl"];
```

Comment:
Why did this not work in the `ConvertToConfig`? And use the `UdpServer.AddressOverrideConfigKey` constant instead.

Reply:
Chicken and the egg situation. 'AddEnvironmentVariables' adds it to the config, but config is never passed through to 'ConvertToConfig'.

```
return config
                .SetBasePath(appPaths.ConfigurationDirectoryPath)
                .AddInMemoryCollection(inMemoryDefaultConfig)
                .AddJsonFile(LoggingConfigFileDefault, optional: false, reloadOnChange: true)
                .AddJsonFile(LoggingConfigFileSystem, optional: true, reloadOnChange: true)
                .AddEnvironmentVariables("JELLYFIN_")
                .AddInMemoryCollection(commandLineOpts.ConvertToConfig());
```